task2-3/src/models/GroupModel.ts

```diff
@@ -0,0 +1,28 @@
+import { Model, DataTypes, InferAttributes, InferCreationAttributes } from 'sequelize';
+import { Permission } from '../types/Permission';
+import { sequelize } from '../loaders/db';
+
+export interface IGroupModel extends Model<InferAttributes<IGroupModel>, InferCreationAttributes<IGroupModel>> {
+  id: string;
+  name: string;
+  permissions: Permission[];
+}
+
+export const GroupModel = sequelize.define<IGroupModel>('group', {
+    id: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        primaryKey: true
+    },
+    name: {
+        type: DataTypes.STRING,
+        allowNull: false
+    },
+    permissions: {
+        type: DataTypes.ARRAY(DataTypes.STRING),
+        allowNull: false
+    }
+}, {
+    timestamps: false
```

Comment:
I like this sequelize feature. It automatically appends `createdAt, updatedAt` for your rows

Reply:
I thought I should turn this feature off, to return the table which consists of fields from the task description)